java-checks-test-sources/src/main/java/checks/RandomizedTestDataCheck.java

```diff
@@ -0,0 +1,45 @@
+package checks;
+
+import java.util.Random;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+
+public class RandomizedTestDataCheck {
+
+  @Test
+  public void randomizedTest() {
+    int userAge = new Random().nextInt(42);  // Noncompliant[[sc=19;ec=31;secondary=11,25,27,29,31,33,42]]{{Replace randomly generated values with fixed ones.}}
+    UUID userID = UUID.randomUUID(); // Noncompliant[[sc=19;ec=36;secondary=12,26,28,30,32,34]]{{Replace randomly generated values with fixed ones.}}
```

Comment:
I don't think it makes sense to include the main location in the secondaries.

Reply:
On the one hand yes.

What I thought about secondary locations is that there are different cases, If we report the main issue on a different level (example class) and highlight secondaries (example method) we do not report the primary one in a secondary.

When we're talking about this issue. Here we report the first occurrence and highlight other usages. So at this point this makes sense to add the first one to secondary locations. And moreover, this way we have a fair counter of all usages.

Wdyt?